StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -36,6 +36,7 @@ extension PaymentSheet {
         case dynamic(String)
         case UPI
         case cashApp
+        case bacsDebit
```

Comment:
Do we need to do this or can we use the STPPaymentMethodType instead?

Reply:
We use this in `performLocalActionsIfNeededAndConfirm` — would it make more sense to check the `confirmParams.paymentMethodParams.type` instead? Not totally familiar with whether that's populated correctly in the deferred flow, so I was trying to follow the pattern I saw in `PaymentMethodType`.

Reply:
That's a good callout for the deferred flow, I'm not sure what our stance on `PaymentMethodType` vs `STPPaymentMethodType` is. I know we want to avoid `dynamic` but that's not being used here.